src/frontend/components/DashboardDocument/index.spec.tsx

```diff
@@ -1,5 +1,6 @@
-import { render, wait } from '@testing-library/react';
+import { fireEvent, render, wait } from '@testing-library/react';
```

Comment:
Maybe the commit title can be clearer
`allow to update document title` => `allow instructors to update document title`